python/lsst/ts/ATHexapod/mock_controller.py

```diff
@@ -0,0 +1,256 @@
+        self.commanded_x = x.decode()
+        self.comanded_y = y.decode()
+        self.commanded_z = z.decode()
+        self.commanded_u = u.decode()
+        self.commanded_v = v.decode()
+        self.commanded_w = w.decode()
+        self.x = x.decode()
+        self.y = y.decode()
+        self.z = z.decode()
+        self.u = u.decode()
+        self.v = v.decode()
+        self.w = w.decode()
+
+    async def referencing_result(self):
+        return (f"X={self.reference_x} "
+                f"Y={self.reference_y} "
+                f"Z={self.reference_z} "
+                f"U={self.reference_u} "
+                f"V={self.reference_v} "
+                f"W={self.reference_w}\n".encode())
+
+    async def reference(self):
+        self.reference_x = 1
+        self.reference_y = 1
+        self.reference_z = 1
+        self.reference_u = 1
+        self.reference_v = 1
+        self.reference_w = 1
+
+    async def target_position(self):
+        return (f"X={self.commanded_x} "
+                f"Y={self.commanded_y} "
+                f"Z={self.commanded_z} "
+                f"U={self.commanded_u} "
+                f"V={self.commanded_v} "
+                f"W={self.commanded_w}\n".encode())
+
+    async def set_low_position_soft_Limit(self, x, y, z, u, v, w):
+        self.minimum_limit_x = x.decode()
+        self.minimum_limit_y = y.decode()
+        self.minimum_limit_z = z.decode()
+        self.minimum_limit_u = u.decode()
+        self.minimum_limit_v = v.decode()
+        self.minimum_limit_w = w.decode()
+
+    async def get_low_position_soft_limit(self):
+        return (f"X={self.minimum_limit_x} " 
+                f"Y={self.minimum_limit_y} "
+                f"Z={self.minimum_limit_z} "
+                f"U={self.minimum_limit_u} "
+                f"V={self.minimum_limit_v} "
+                f"W={self.minimum_limit_w}\n".encode())
+
+    async def set_high_position_soft_limit(self, x, y, z, u, v, w):
+        self.maximum_limit_x = x.decode()
+        self.maximum_limit_y = y.decode()
+        self.maximum_limit_z = z.decode()
+        self.maximum_limit_u = u.decode()
+        self.maximum_limit_v = v.decode()
+        self.maximum_limit_w = w.decode()
+
+    async def get_high_position_soft_limit(self):
+        return (f"X={self.maximum_limit_x} "
+                f"Y={self.maximum_limit_y} "
+                f"Z={self.maximum_limit_z} "
+                f"U={self.maximum_limit_u} "
+                f"V={self.maximum_limit_v} "
+                f"W={self.maximum_limit_w}\n".encode())
+
+    async def on_target(self):
+        pass
+
+    async def get_position_unit(self):
+        pass
+
+    async def offset(self, x, y, z, u, v, w):
+        pass
+
+    async def check_offset(self, x, y, z, u, v, w):
+        pass
+
+    async def set_pivot_point(self, x, y, z):
+        self.pivot_x = x.decode()
+        self.pivot_y = y.decode()
+        self.pivot_z = z.decode()
+
+    async def getPivotPoint(self):

```

Comment:
Picky, but please use snake_case to match the rest of the code. Consider `format_pivot_point`.